OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+    ```
+    **Note**: 
+    - If you want to download image from the Docker store, see [FAQ](#7-alternate-download-location-for-oracle-fusion-middleware-infrastructure-and-oracle-database-images) section.
+    - If you want to build image from the GitHub, see [FAQ](#8-how-do-i-build-an-oracle-database-1221x-base-image) section.
+	
+	
+### C. To download Oracle WebCenter Sites Docker files and binary file.
+
+1. Download the [docker-images-master](https://github.com/oracle/docker-images/archive/master.zip) zip file. The zip file contains Docker files and samples to build Docker images for Oracle products and open source projects.
+2. Download Oracle WebCenter Sites 12c R2 12.2.1.3 binary from [OTN](http://www.oracle.com/technetwork/middleware/webcenter/sites/downloads/index.html).
+3. Save the Oracle WebCenter Sites 12.2.1.3 binary at this location: `../docker-images-master/OracleWebCenterSites/dockerfiles/12.2.1.3/`.
+
+## 5. Building Oracle WebCenter Sites Docker Images
+1. Go to the folder where Docker scripts are located:
+```
+    $ cd ../docker-images-master/OracleWebCenterSites/dockerfiles
+```
+
+2. To build Oracle WCS Docker image, run the following command:
+```
+   $ sh buildDockerImage.sh -v 12.2.1.3
+```
+   For more information on usage of <i>buildDockerImage.sh</i> command, see [FAQ](#1-what-is-the-usage-of-builddockerimagesh-file),
+
+**IMPORTANT**: The resulting image has the automated scripts to:
+-  Run RCU
+-  Create and configure a WebLogic domain
+-  Run WebCenter Sites Configuration process while creating an admin container out of the image
+
+## 6. Preparing to Run Oracle WebCenter Sites Docker Image
+Before running the Oracle WebCenter Sites Docker image, you need to set up a communication network between containers on the same Docker engine’s host. You also need to set up data volume to enable file sharing between Docker containers and database container.
+
+Configure an environment to run the WebCenter Sites Docker image by following these steps:
+
+##### A. Creating a user-defined network
+##### B. Mounting host directory as a data volume 
+##### C. Setting up an Oracle Database Docker container
+
+### A. Creating a user-defined network
+Create a user-defined network to enable containers to communicate by running the following command:
+```
+   $ docker network create -d bridge <network_name>
+```
+Sample command:
+```
+   $ docker network create -d bridge WCSitesNet
+```
+### B. Mounting host directory as a data volume
+You need to mount volumes, which are directories stored outside a container's file system, to store database data files and WLS domain files. The default location of the volume in the container is "/var/lib/docker/volumes". 
+
+This option lets you mount a directory from your Docker engine’s host to a container as volume. This volume is used to store database data files and WebLogic server domain files. The volume is created at this location <i>/scratch/DockerVolume/WCSitesVolume/</i>.
```

Comment:
Remove "Docker engine's" and replace \<i\> and \<\/i\> with backticks so that the path is monospaced.